dev-libs/json_c/json_c-0.13.recipe

```diff
@@ -0,0 +1,71 @@
+SUMMARY="A JSON implementation in C"
+DESCRIPTION="JSON-C implements a reference counting object model that allows \
+you to easily construct JSON objects in C, output them as JSON formatted \
+strings and parse JSON formatted strings back into the C representation of \
+JSON objects."
+HOMEPAGE="https://github.com/json-c/json-c/wiki"
+COPYRIGHT="2009-2012 Eric Haszlakiewicz
+	2004, 2005 Metaparadigm Pte Ltd"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://github.com/json-c/json-c/archive/json-c-$portVersion.tar.gz"
+CHECKSUM_SHA256="259e3a81a804246b128fba7117ab6d261314b33e794cc4d2db71a0b59690d18b"
+SOURCE_DIR="json-c-json-c-$portVersion"
+
+ARCHITECTURES="?x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	json_c$secondaryArchSuffix = $portVersion
+	lib:libjson_c$secondaryArchSuffix = 4.0.0 compat >= 4
```

Comment:
Shall we add a `libVersion` and its `libVersionCompat`?
```diff
 ARCHITECTURES="?x86_gcc2 x86 x86_64"
 SECONDARY_ARCHITECTURES="x86"
+
+libVersion="4.0.0"
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
+
 PROVIDES="
 	json_c$secondaryArchSuffix = $portVersion
-	lib:libjson_c$secondaryArchSuffix = 4.0.0 compat >= 4
+	lib:libjson_c$secondaryArchSuffix = $libVersion
```
(Of course, the purpose of this change is to replace all other occurences of `4.0.0 compat >= 4` by `$libVersionCompat`.)

BTW, you can probably remove `json_c-0.12.1.recipe` because there is only one recipe which needs `lib:libjson_c`, and that's `sys-auth/yubikey-personalization/libykpers-1.18.0.recipe`, which is not critical.
So, if `libykpers-1.18.0.recipe` builds fine with `json_c-0.13.recipe` then it would be nice to include in this PR the remval of `json_c-0.12.1.recipe`. Thanks!